lblod/frontend-gelinkt-notuleren
# GN-4692: Enable `BesluitTopicPlugin`

### Overview

Enabble `BesluitTopicPlugin` plugin allowing to connect multiple topics (`BesluitThema`) to a decision (besluit).

##### connected issues and PRs:

* https://binnenland.atlassian.net/browse/GN-4692
* https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/426

### Setup

1. Checkout
2. Run

### How to test/reproduce

Please repeat after me, see the video :) 


https://github.com/lblod/frontend-gelinkt-notuleren/assets/769698/71351d07-ea72-498c-8be1-7afd1b48b2af




### Challenges/uncertainties

1. UI pill is kinda meh 🤷🏻‍♂️ 



### Checks PR readiness
